challenges/objects-arrays.js

```diff
@@ -71,9 +71,21 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = graduates.map(function cb(i){
-  return i.university;
-}).sort();
+
+//const universities = graduates.map(function cb(i){
+//  return i.university;
+//}).sort(); ^-- I'll learn to read first, one day
+
+const universities = [];
+
+function univs () {
+  for (i=0; i<graduates.length; i++) {
+    universities.push(graduates[i].university);
```

Comment:
It looks like you noticed based on your comments in lines 75-77, but you don't have to create functions to run loops!